android/src/main/java/com/wultra/android/malwarelytics/reactnative/AVObjects.kt

```diff
@@ -22,11 +22,32 @@ import com.wultra.android.antimalware.ApkThreat
 import com.wultra.android.antimalware.MalwareFlag
 import com.wultra.android.antimalware.smart.SmartProtectionResult
 import com.wultra.android.antimalware.update.*
+import com.wultra.android.appprotection.utils.AppInfo
 import com.wultra.android.rasp.*
+import java.io.ByteArrayOutputStream
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.Bitmap
+import android.util.Base64
 
 // This file contains functions that converts various objects from native Malwarelytics SDK
 // to the JavaScript objects.
 
+fun AppInfo.toJs(): ReadableMap {
+    val iconString = (icon as? BitmapDrawable)?.let { bd ->
```

Comment:
Well, there can be other types of drawables there. 
For example VectorDrawable and many others.
I guess that the only solution is to convert to bitmap via drawing into Canvas.

Something like:
```
fun drawableToBitmap(drawable: Drawable): Bitmap {
    val width = drawable.intrinsicWidth.takeIf { it > 0 } ?: 128
    val height = drawable.intrinsicHeight.takeIf { it > 0 } ?: 128
    val bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
    val canvas = Canvas(bitmap)
    drawable.setBounds(0, 0, canvas.width, canvas.height)
    drawable.draw(canvas)
    return bitmap
}
```